utils/oscap.c

```diff
@@ -117,6 +120,7 @@ static int print_versions(const struct oscap_action *action)
 	printf("CPE Version: %s\n", cpe_dict_model_supported());
 	printf("CVSS Version: %s\n", cvss_model_supported());
 	printf("CVE Version: %s\n", cve_model_supported());
+	printf("CVRF Version: %s\n", cvrf_model_supported());
```

Comment:
Add CVRF to the bottom. It's not a SCAP standard, so let's not interleave it with the SCAP standards.